### DISCUSSION TITLE
Reduce number in comments added by preview action

### DISCUSSION THREAD

**Discussion by Czaki** [chanzuckerberg/napari-hub#448] [Ideas]
An annoying feature of `napari-hub-preview-action` is that every pus action adds the same looking comment. For example:

>  Preview page for your plugin is ready here:
>  [preview.napari-hub.org/4DNucleome/PartSeg/554](https://preview.napari-hub.org/4DNucleome/PartSeg/554)

But other actions like [codecov](https://github.com/marketplace/codecov) or [sourcery-ai](https://github.com/marketplace/sourcery-ai) solve this problem by editing the existing comment or removing the old one. As napari hub does not update the link then maybe one comment per PR could be enough?

**Comment by neuromusic** [comment]
That's @Czaki ... we're going to be tackling some bugs/improvements in the coming weeks and this one is high priority. Tracking this in https://github.com/chanzuckerberg/napari-hub/issues/422
